packages/odyssey-storybook/src/components/odyssey-mui/Status/Status.stories.tsx

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { Story } from "@storybook/react";
+import { Chip } from "@okta/odyssey-react-mui";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+
+import StatusMdx from "./Status.mdx";
+
+export default {
+  title: `MUI Components/Status`,
+  component: Chip,
+  parameters: {
+    docs: {
+      page: StatusMdx,
+    },
+  },
+  argTypes: {
+    label: {
+      control: "text",
+      defaultValue: "Warp drive in standby",
+    },
+    severity: {
+      control: "radio",
+      options: ["default", "error", "info", "success", "warning"],
+      defaultValue: "default",
+    },
+  },
+  decorators: [MuiThemeDecorator],
+};
+
+const DefaultTemplate: Story = (args) => {
+  return <Chip label={args.label} variant="status" color={args.severity} />;
+};
```

Comment:
Weren't we going to wrap this as `<Status />`?

Is this PR a v1 or was the intention to use `variant="status"` instead of a separate component wrapped by `Status`?

Reply:
PR is a v1. I haven't wrapped a component yet, so I wanted to get the theme work in as normal/previously, then start the wrapping work in a fresh branch.

(Apologies if this is sending a bunch of notifications, @KevinGhadyani-Okta. Getting some weird GH behavior right now.)